src/server/chromium/crPage.ts

```diff
@@ -197,6 +197,7 @@ export class CRPage implements PageDelegate {
   }
 
   async closePage(runBeforeUnload: boolean): Promise<void> {
+    await this._mainFrameSession._stopScreencast();
```

Comment:
Call this from _didClose as it doesn't send any messages to the browser?

Reply:
It is already so